src/ui/components/Tour/Tour.module.css

```diff
@@ -21,16 +21,24 @@
 }
 
 .stepCallout {
-  border: 1px solid #fff;
-  background-color: rgba(255, 255, 255, 0.05);
-  padding: 0.5em 0.5em;
-  border-radius: 4px;
+  display: flex;
+  align-items: top;
+  color: #ff0;
+}
+
+.stepCallout .Icon {
+  width: 16px;
+  color: #ff0;
```

Comment:
⚠️ Hard-coded color

Reply:
This is appropriate because the Tour does not change in different themes. It's always this color.

I understand the argument that everything should have a variables and we can just set the variable to the same color for both dark and light theme, but I disagree with it.

Reply:
That's kind of an implementation detail and could be broken by a change to either theme– which would not know to scan for hard-coded colors like this. We really shouldn't be adding hard-coded colors to our styles.

Reply:
I'll change it, but it's important to understand this is a difference of opinion, not objective fact.

This CSS sets an animating background to slightly fade between a dark purple and a deep blue. These are hard-coded to those bgcolor values, and I chose to set the color value the same way. This makes the component and the css self-encapsulating. Theme changes wouldn't affect how this component renders, and I believe there is a benefit to describing single values once rather than having to hunt it down in two places in variables.css.

But happy to disagree and change to your preference here.

Reply:
Ok @bvaughn I've added --tour-body-color and --tour-callout-body-color to address these